scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -383,12 +389,17 @@ class FossId(
         return results
     }
 
-    private suspend fun List<Scan>.findLatestPendingOrFinishedScan(url: String, revision: String? = null): Scan? =
+    private suspend fun List<Scan>.findLatestPendingOrFinishedScan(
```

Comment:
Minor nit: The order of parameters here is different than to the `createScan()` function. It would probably make sense to align it everywhere.